conans/client/graph/python_requires.py

```diff
@@ -54,7 +54,12 @@ def __getitem__(self, item):
         try:
             return self._pyrequires[item]
         except KeyError:
-            raise ConanException("'%s' is not a python_require" % item)
+            # https://github.com/conan-io/conan/issues/8546
```

Comment:
Sounds like this class could get a refactor, but lets wait until moving this forward first, if we think it is correct.